CONTRIBUTING.md

```diff
@@ -244,12 +244,24 @@ This workflow then files a PR to `master` with the rendered and live files.
 
 ## Cheatsheets
 
-Training modules have corresponding cheatsheets in `module-cheatsheets`. 
-Cheatsheets are written in plain markdown and can be converted to a shareable PDF format interactively as follows:
+Training modules have corresponding cheatsheets in `module-cheatsheets`.
+When choosing documentation links to incorporate in cheatsheets, we prefer to use [`https://rdrr.io/`](https://rdrr.io/) when possible for Base R and Bioconductor, and we prefer to use [`https://www.tidyverse.org/`](https://www.tidyverse.org/) for `tidyverse` functions.
 
-+ Be on a MacOS computer
-+ Install and/or open the [MacDown program](https://macdown.uranusjr.com/)
-+ Open the markdown cheatsheet in MacDown, and go to `File` -> `Export` -> `PDF`
-+ Save appropriately and voila!
+Cheatsheets are written in plain markdown and are converted to a shareable PDF format using the `npm` library [`mdpdf`](https://github.com/BlueHatbRit/mdpdf), with the default PDF style.
+You can install this package globally for all users with (or remove the `-g` for local installation) with:
 
-When choosing documentation links to incorporate in cheatsheets, we prefer to use [`https://rdrr.io/`](https://rdrr.io/) when possible for Base R and Bioconductor, and we prefer to use [`https://www.tidyverse.org/`](https://www.tidyverse.org/) for `tidyverse` functions. 
\ No newline at end of file
+```
+npm install -g mdpdf
+```
+
+In addition, cheatsheet table of contents are created with the `npm` library [`doctoc`](https://github.com/thlorenz/doctoc), which can similarly be globally installed with:
```

Comment:
I personally am ok with this, _but_ it was my understanding that we were trying to avoid extensions for generating cheatsheets (see my previous attempt to use a VS Code extension for PDF rendering..) and use more standalone/command line tools.

Reply:
I see a difference between "this may be a convenient alternative with the same effect" and "this is the way it has to be done." If we do make this workflow an action, we can still code all the steps to be sure they are run, with no requirement of VScode.

Reply:
I definitely see that, but now I'm a little confused on how I should proceed! Are you thinking something like one of these routes? 
- Keep current CONTRIBUTING content, but _also_ suggest that "Markdown All in One" could be used in place of `doctoc` if one were so moved. 
- Keep `mdpdf` but remove `doctoc` and _instead_ recommend that the use of "Markdown All in One" (which I just looked up, does HTML but not PDF conversion) for TOCs
- Focus on the "convenient alternatives" aka VS code extensions for both TOC and PDF rendering. Then later if we want to include an action for rendering, we can write a script to use `doctoc` and `mdpdf` or similar.

Reply:
Done in https://github.com/AlexsLemonade/training-modules/pull/700/commits/955201b7c993b1c11375324d3bfd39cffee6fef1.